pages/overview.vue

```diff
@@ -0,0 +1,163 @@
+    TableOfContents,
+    ContentSection,
+    TheQuickStart,
+    AppCta
+  },
+  head () {
+    return {
+      title: 'Qiskit Overview'
+    }
+  }
+})
+export default class extends QiskitPage {
+  routeName = 'overview'
+
+  contentSections = [
+    {
+      id: 'applications',
+      title: 'Research Applications',
+      description: 'Qiskit allows for easy research and development for specific industry use cases that have the highest potential for quantum advantage.',
+      link: {
+        label: 'See docs',
+        url: '/'
+      }
+    },
+    {
+      id: 'algorithms',
+      title: 'Collection of Algorithms',
+      description: 'Qiskit contains a generic framework of cross-domain quantum algorithms upon which applications for near-term quantum computing can be built.',
+      link: {
+        label: 'See docs',
+        url: '/'
+      }
+    },
+    {
+      id: 'characterization',
+      title: 'Experimentalist Toolbox',
+      description: 'Qiskit provides a framework for understanding and mitigating noise in quantum circuits and systems. The experiments provided in Qiskit are grouped into the topics of characterization, verification and mitigation.',
+      link: {
+        label: 'See docs',
+        url: '/'
+      }
+    },
+    {
+      id: 'circuits',
+      title: 'Circuits',
+      description: 'Qiskit provides a set of tools for composing quantum programs at the level of circuits and pulses, optimizing them for the constraints of a particular physical quantum processor, and managing the batched execution of experiments on remote-access backends.',
+      link: {
+        label: 'See docs',
+        url: '/'
+      }
+    },
+    {
+      id: 'simulators',
+      title: 'Simulate Quantum Hardware',
+      description: 'Qiskit provides a framework for understanding and mitigating noise in quantum circuits and systems. The experiments provided in Qiskit are grouped into the topics of characterization, verification and mitigation.',
+      link: {
+        label: 'See docs',
+        url: '/'
+      }
+    },
+    {
+      id: 'hardware',
+      title: 'Run Circuits on Real Quantum Systems',
+      description: 'Circuits are the foundational roots for our software stack. Qiskit provides a set of tools for composing quantum programs at the level of circuits and pulses, optimizing them for the constraints of a particular physical quantum processor, and managing the batched execution of experiments on remote-access backends. Qiskit is modularly constructed, simplifying the addition of extensions for circuit optimizations and backends.',
+      link: {
+        label: 'See docs',
+        url: '/'
+      }
+    }
+  ]
+}
+</script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
+
+.overview-page {
+  background-color: white;
+
+  &__content-container {
+    @include contained();
+    display: flex;
+    align-items: flex-start;
+    overflow: visible;
+    margin-bottom: $layout-07 / 2;
```

Comment:
Please, double-check with @JRussellHuffman if we can stick with Carbon layout spacing rules and convert this into a `layout-06` or `layout-05`.

Reply:
During sprint review, we decided this would be `layout-05`.